doc/release/1.7.0-notes.rst

```diff
@@ -91,13 +328,637 @@ Backwards incompatible changes
 Other changes
 =============
 
+We now accept and leverage performance improvements from the ahead-of-time
+Python-to-C++ transpiler, Pythran, which can be optionally disabled but is
+enabled by default at build time.
+
+There are two changes to the default behavior of `scipy.stats.mannwhitenyu`:
+
+- For years, use of the default ``alternative=None`` was deprecated; explicit
+``alternative`` specification was required. Use of the new default value of
+``alternative``, "two-sided", is now permitted.
+- Previously, all p-values were based on an asymptotic approximation. Now, for
+  small samples without ties, the p-values returned are exact by default.
+
+Support has been added for PEP 621 (project metadata in ``pyproject.toml``)
+
+We now support a Gitpod environment to reduce the barrier to entry for SciPy
+development.
```

Comment:
Can you cross-link this to this page via a label and `:ref:`: http://scipy.github.io/devdocs/dev/contributor/quickstart_gitpod.html ?